### PR TITLE
Add notes for GC-07-28 meeting

### PR DESCRIPTION
**Pull request by binji**

### CODE REVIEW

**Comment by binji** [comment]
@tebbi Please add a link to your slides

File: gc/2020/GC-07-28.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+LW: can it be useful for JS to create/consume GC objects even without Wasm around. Just wanting to use these as objects that are more JIT optimizable, maybe want stricter typing.
+
+TT: I would like that. I think that's missing from JS language. Can't comment on how difficult it would be to get through ES, but I would like it.
+
+RT: like the typed objects paper
+
+LW: if it was in JS, a first class extension to the class syntax to declare this with extra keywords… when you think about that, how does that interact with Wasm, use that only, or with Wasm?
+
+BS: probably a good place to leave it. Thanks TT. Continue discussion on GitHub issue, open a new one if no existing.
```
Comment (chicoxyzzy) [inline gc/2020/GC-07-28.md:189]: a link to the issue would be helpful!

**Comment by tebbi** [comment]
Issue: https://github.com/WebAssembly/gc/issues/107
Slides: https://docs.google.com/presentation/d/1BqRlDrQIYdkRHHtoZ7F8a2tpFwYoumCCbqQ4SpULajI/

File: gc/2020/GC-07-28.md (hunk 1 above)
Comment (chicoxyzzy) [inline gc/2020/GC-07-28.md:189]:
```suggestion
BS: probably a good place to leave it. Thanks TT. Continue discussion on [GitHub issue](https://github.com/WebAssembly/gc/issues/107).
```

File: gc/2020/GC-07-28.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+* Sabine
+* Slava Kuzmich
+* Tobias Tebbi
+* Wouter Van Oortmersson
+* Zalim Basharov
+* Zhi An Ng
+
+### Discussion: Wasm interop with host languages like JavaScript (@tebbi)
+
+TODO(tebbi): slides
```
Comment (chicoxyzzy) [inline gc/2020/GC-07-28.md:56]:
```suggestion
Slides: https://docs.google.com/presentation/d/1BqRlDrQIYdkRHHtoZ7F8a2tpFwYoumCCbqQ4SpULajI/
Issue: https://github.com/WebAssembly/gc/issues/107
```

**Pull request merged**